modules/date.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+import { DateTime } from './luxon.js';
+
+export default function luxon() {
+  const dt = DateTime.now();
+  return dt.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}
```

Comment:
- [ ] NPM is a tool that allows us to handle external packages and import them into our project. In this case, we can run `npm install luxon` on the command line to bring the library to our project. Then we can import this one as follows `import { DateTime } from 'luxon'. NPM is programmed to find external libraries in the `node_modules` folder when we don't specify the relative paths, eg: `./`, `../`, etc.

If you have installed a package correctly you should see it listed on the dependencies of the `package.json` file.

[Read more about working with NPM here.](https://www.freecodecamp.org/news/what-is-npm-a-node-package-manager-tutorial-for-beginners/)

Reply:
Great! Thank you for working us through the process! We implemented successfully